oat-sa/tao-item-runner-fe
# Move to github actions and update dependencies

- Update the dependencies
 - Move from Jenkins to GH actions (with conv commit check, code coverage report and eslint check in the code)
 - Fix eslint errors

How to test:
 - run unit tests
 - run the code coverage
 - run the lint scripts
 - check the github action in this PR
